turbo/cli/make_app.go

```diff
@@ -24,5 +33,99 @@ func MakeApp(action func(*cli.Context), cliFlags []cli.Flag) *cli.App {
 		debug.Exit()
 		return nil
 	}
+	app.Commands = []cli.Command{initCommand}
 	return app
 }
+
+var initCommand = cli.Command{
+	Action:    MigrateFlags(initGenesis),
+	Name:      "init",
+	Usage:     "Bootstrap and initialize a new genesis block",
+	ArgsUsage: "<genesisPath>",
+	Flags: []cli.Flag{
+		utils.DataDirFlag,
+	},
+	Category: "BLOCKCHAIN COMMANDS",
+	Description: `
+The init command initializes a new genesis block and definition for the network.
+This is a destructive action and changes the network in which you will be
+participating.
+
+It expects the genesis file as argument.`,
+}
+
+// initGenesis will initialise the given JSON format genesis file and writes it as
+// the zero'd block (i.e. genesis) or will fail hard if it can't succeed.
+func initGenesis(ctx *cli.Context) error {
+	// Make sure we have a valid genesis JSON
+	genesisPath := ctx.Args().First()
+	if len(genesisPath) == 0 {
+		utils.Fatalf("Must supply path to genesis JSON file")
+	}
+
+	file, err := os.Open(genesisPath)
+	if err != nil {
+		utils.Fatalf("Failed to read genesis file: %v", err)
+	}
+	defer file.Close()
+
+	genesis := new(core.Genesis)
+	if err := json.NewDecoder(file).Decode(genesis); err != nil {
+		utils.Fatalf("invalid genesis file: %v", err)
+	}
+
+	// Open and initialise both full and light databases
+	stack := MakeConfigNodeDefault()
+	defer stack.Close()
+
+	for _, name := range []string{"chaindata", "lightchaindata"} {
```

Comment:
I think you don't need this loop. Because variable `name` not used anywhere and Erigon has no `lightchaindata`